modules/media.js

```diff
@@ -119,11 +127,16 @@ Media.prototype.backup = function(type, oncomplete) {
  * @param {callback} oncomplete
  */
 Media.prototype.restore = function(type, oncomplete) {
+
   var paths = ffosbr.settings.getBackupDirectoryPaths();
+  var empty = true; // default to true, fetching a file will set it false
 
-  ffosbr.media.get('sdcard1', paths[type], function(file) {
-    if (!file) {
-      return;
+  ffosbr.media.get('sdcard1', paths[type], function(file, error) {
+
+    if (error || !file) {
+      return oncomplete(type, error);
```

Comment:
Again, we'll call `oncomplete(type, error)` if `error` is undefined and `!file` evaluates to `true`